src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -254,8 +308,8 @@ public string GenerateBundle(IReadOnlyList<FileSpec> fileSpecs)
                     using (FileStream file = File.OpenRead(fileSpec.SourcePath))
                     {
                         FileType targetType = Target.TargetSpecificFileType(type);
-                        long startOffset = AddToBundle(bundle, file, targetType);
-                        FileEntry entry = BundleManifest.AddEntry(targetType, relativePath, startOffset, file.Length);
+                        (long startOffset, long compressedSize) = AddToBundle(bundle, file, targetType);
+                        FileEntry entry = BundleManifest.AddEntry(targetType, relativePath, startOffset, file.Length, compressedSize, Target.BundleMajorVersion);
                         Tracer.Log($"Embed: {entry}");
```

Comment:
Nit: can we also add info about whether the file was compressed in the `ToString` override of `FileEntry`?

Reply:
Done